benchmarks/infill_density/infill_ascii.prm

```diff
@@ -0,0 +1,199 @@
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, single Stokes
+set CFL number                             = 0.1
+set Maximum time step                      = 750
+set Output directory                       = output
+set Pressure normalization                 = no
+
+# Model geometry (500x500 km, 50 km spacing)
+subsection Geometry model
+  set Model name = box
+
+  subsection Box
+    set X repetitions = 15
+    set Y repetitions = 30
+    set X extent      = 1200e3
+    set Y extent      = 1200e3
+  end
+end
+
+# Mesh refinement specifications
+# Refine the mesh in the uppermost part of the model
+# to better capture lithospheric response to the load.
+subsection Mesh refinement
+  set Initial adaptive refinement              = 2
+  set Initial global refinement                = 1
+  set Time steps between mesh refinement       = 1
+  set Minimum refinement level                 = 0
+  set Strategy                                 = minimum refinement function, strain rate
+  subsection Minimum refinement function
+    set Coordinate system                      = cartesian
+    set Variable names                         = x,y,t
+    set Function expression                    = if(y>=1050e3, 3, 0)
+  end
+end
+
+# Element types
+subsection Discretization
+  set Composition polynomial degree           = 2
+  set Stokes velocity polynomial degree       = 2
+  set Temperature polynomial degree           = 1
+  set Use locally conservative discretization = true
+  set Use discontinuous temperature discretization = false
+  set Use discontinuous composition discretization = true
+  subsection Stabilization parameters
+      set Use limiter for discontinuous composition solution = true
+      set Global composition maximum =  1.e13,  1.e13,  1.e13, 1.0
+      set Global composition minimum = -1.e13, -1.e13, -1.e13, 0.0
+  end
+end
+# Formulation classification
+subsection Formulation
+  set Enable elasticity = true
+end
+
+subsection Mesh deformation
+  set Mesh deformation boundary indicators = top: free surface
+  set Additional tangential mesh velocity boundary indicators = left, right
+  subsection Free surface
+    set Surface velocity projection = vertical
+  end
+end
+
+# Velocity boundary conditions
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = bottom, left
+end
+
+# Prescribe a fixed vertical traction on the top boundary
+subsection Boundary traction model
+  set Prescribed traction boundary indicators = top y: infill ascii data, right: initial lithostatic pressure
+end
+
+subsection Boundary traction model
+  subsection Ascii data model
+    set Data directory = $ASPECT_SOURCE_DIR/benchmarks/infill_density/data/
+    set First data file model time = 0
+    set Data file name = box_2d_%s.%d.txt
+    set Data file time step = 750
+  end
+  subsection Infill ascii
+    set Rock density = 2000
+    set Sediment density = 2000
```

Comment:
Out of curiosity, why does the GMT benchmark assume the rock and sediment density to be equal?

Reply:
I honestly dont 100% understand why this is the case, but apparently the fourier solution is only exact with a constant density contrast between the load and the infill. If the infill varies then the fourier solution is still good but it is no longer 'exact' and now has to be approximated with a scaling prefactor.